libmscore/chord.cpp

```diff
@@ -3569,8 +3569,8 @@ void Chord::setNoteEventLists(QList<NoteEventList>& ell)
       {
       if (notes().empty())
             return;
-      Q_ASSERT(ell.size() == notes().size());
-      for (size_t i = 0; i < ell.size(); i++) {
+      Q_ASSERT(ell.size() == int(notes().size()));
+      for (size_t i = 0; int(i) < ell.size(); i++) {
```

Comment:
Why not to declare `i` as `int` instead? Then there will be no need in explicit `int()` conversions, both here and in the next (3574) line.

Reply:
because `notes.size()` correctly reeturn `size_t` and also this is the corryct type for array indices

Reply:
Actually it could/should be seen as a workaround for a Qt bug. Question might be whether to cast `QList::size()`to `size_t` instead. But MSVC still wants the cast to int in the line below, so that seems the more consistent way.